tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -57,7 +74,7 @@ def _impl(ctx):
       "--closure_entry_point=%s" % ctx.attr.main,
       "--js_output_file=%s" % ctx.outputs.out.path,
       "--language_in=ECMASCRIPT5_STRICT",
-      "--manage_closure_dependencies",
+      "--dependency_mode=LOOSE",
```

Comment:
Good catch. I missed the deprecated `--closure_entry_point` argument on the other line.